java/test/TestH5Fbasic.java

```diff
@@ -213,29 +213,6 @@ public void testH5Fclose()
         }
     }
 
-    @Test(expected = HDF5LibraryException.class)
-    public void testH5Fclose_twice() throws Throwable
```

Comment:
Can you just leave this in as an ignore? We shouldn't delete a test just to make the tests pass and the FFM bindings should pass the same tests as the JNI bindings anyway.